hello_world_on_chain/lib/TAParserLib.sol

```diff
@@ -1,23 +1,21 @@
 //  SPDX-License-Identifier: MIT
 pragma solidity ^0.8.10;
 
-import {JsmnSolLib} from "./JsmnSolLib.sol";
```

Comment:
I only made these changes to the version in hello world.  If we decide to go with this direction, we can add these changes to the other repos.